templates/nginx_values.yaml

```diff
@@ -9,10 +9,6 @@ controller:
       service.beta.kubernetes.io/aws-load-balancer-target-node-labels: kubernetes.io/os=linux
       service.beta.kubernetes.io/aws-load-balancer-type: "nlb"
   metrics:
-    enabled: true
-    serviceMonitor:
-      enabled: true
-      additionalLabels:
-        release: "prometheus" 
+    enabled: false

```

Comment:
@amarin-dspace I cannot assess if this change will break something

Reply:
In this case to have it configurable by a variable would be the better variant

Reply:
Reverted the last commit and set default value to 'false' in the variable.